composer.json

```diff
@@ -46,7 +46,7 @@
         "phpstan/phpstan": "^1.2.0",
         "symfony/cache": "^5.4",
         "symfony/phpunit-bridge": "^6.0",
-        "symfony/validator": "^3.4.30|^4.3|^5.0",
+        "symfony/validator": "^3.4.30|^4.3|^5.0|^6.0",
```

Comment:
Cf https://symfony.com/releases, I will tend to restrict to "^4.4.30|^5.4|^6.0" for all Sf deps.

Reply:
Sure - that means we drop support for Symfony 3.* ? I'm ok with it, but I'm wondering if this should trigger a major release or not... WDYT ?

Reply:
Fact is there are no deprecation notices to fix https://github.com/a2lix/TranslationFormBundle/runs/4716968578?check_suite_focus=true#step:9:101 with highest deps that could be the opportunity to drop some old deps & code with a major release.
So.. maybe merge as your initial PR finally.

Reply:
So if I understand correctly: merge this PR as-is, and then do another PR to remove old deps (Sf3.4) ?

Reply:
Merge this as-is yes.
And maybe we postpone the removal of old SF versions when it will be really required by code changes and so major release.
I don't have definitive opinion on this, except that I'm not in favor to release a major release to only drop some SF versions.
We could go with `^4.4.30|^5.3|^6.0` in a minor release but while the CI doesn't complain with current lower deps, maybe it's not a good idea?..

Reply:
Hello,

Did you figured out the best way to tag ? I'd prefer a major version as it's safer but I understand if you prefer not (but you're the last lib I need to be sf6 compliant so 😅).

Thank you ;)

Reply:
Same here 😅